Blog-projects/Widget-window-in-Qt-Quick-app/fontcontrolswidgetsform.h

```diff
@@ -0,0 +1,40 @@
+#ifndef FONTCONTROLSWIDGETSFORM_H
+#define FONTCONTROLSWIDGETSFORM_H
+
+#include <QWidget>
+
+namespace Ui {
+class FontControlsWidgetsForm;
+}
+
+class FontControlsWidgetsForm : public QWidget
+{
+    Q_OBJECT
+    Q_PROPERTY(QString text READ text WRITE setText NOTIFY textChanged)
+    Q_PROPERTY(QString font READ font NOTIFY fontChanged)
+
+public:
+    explicit FontControlsWidgetsForm(QWidget *parent = nullptr);
+    ~FontControlsWidgetsForm();
+
+    const QString text();
+    void setText(const QString&);
+    QString font() const;
```

Comment:
fontFamily(), rather?

font() sounds like it will return a QFont....

Reply:
Switched to using fontFamily where applicable.